pkg/cloudprovider/provider/openstack/provider.go

```diff
@@ -480,8 +481,10 @@ func (p *provider) Validate(spec clusterv1alpha1.MachineSpec) error {
 	}
 
 	// Required fields
-	if _, err := getRegion(client, c.Region); err != nil {
-		return fmt.Errorf("failed to get region %q: %v", c.Region, err)
+	if !strings.Contains(c.IdentityEndpoint, "auth.cloud.ovh.net") {
```

Comment:
Please create a const for "auth.cloud.ovh.net" and move it to the top of the file. That will help in visibility.

Reply:
ok, thanks! :)